frontend/src/Components/Results/Graphs/MultiBarChart.tsx

```diff
@@ -0,0 +1,34 @@
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend,
+} from "chart.js";
+import { Bar } from "react-chartjs-2";
+
+ChartJS.register(CategoryScale, LinearScale, PointElement, LineElement, BarElement, Title, Tooltip, Legend);
+
+interface MultiBarChartProps {
+  xAxis: any[];
+  data: any[];
+}
+// Y axis number of messages
+// X axis agent type
+// let data contain the values (y-axis) and titles (message type)
+// i.e. [{label1: [], data1: []}, ...]
```

Comment:
Can remove, the component is general for any data to be plotted!

Reply:
yeah oops that was just my notes xd